draft-ietf-quic-multipath.md

```diff
@@ -318,8 +318,11 @@ Path ID 0.
 
 After the handshake concluded if negotiation of multipath support succeeded,
 endpoints SHOULD use PATH_ACK frames instead of ACK frames,
-also for acknowledging so far unacknowledged 0-RTT packets, using
-Path ID 0. Similarly after a successful handshake, endpoints SHOULD also use
+including for acknowledging so far unacknowledged 0-RTT packets using Path ID 0.
+However, even after successful negotiation, endpoints MAY use ACK frames
+to acknowledge 0-RTT packets or 1-RTT packets received on initial path with Path ID 0,
+and these ACK frames need to be processed correctly by the peer.
```

Comment:
Sorry, I merged the nits and that made the conversation disappear here, so I'm repost my last comment and change proposal:

The point of adding this sentence is to make the receiver side aware and not to specify any requirements for the sender. There the sentence is rather stating a fact (like "note, this can happen and is valid") that setting a normative protocol requirement. If we want to phrase a requirement here, it should be on the receiver (as Quentin also says below). So basically saying something like:

Endpoints MUST process ACK frames that acknowledge 0-RTT packets or 1-RTT packets for Path ID 0, even after successful negotiation of this extension.

Further the intention of adding something here as discussed in the issue was to give a bit of a reasoning why there is a SHOULD in the above sentence and why it could be useful to still send ACK frames. So I propose to further add a sentence like this:

For example, ACK frames might be preferred by the sender as long as only the initial path with Path ID 0 is in use.

Reply:
```suggestion
Endpoints MUST process ACK frames that acknowledge 0-RTT packets or 1-RTT packets
for Path ID 0, even after successful negotiation of this extension. For example, ACK frames
might be preferred by the sender as long as only the initial path with Path ID 0 is in use.
```

Reply:
+1. This is a good change.

Reply:
Agree too and committed suggestion.